pkg/kubernetes/views/registry-dashboard-page.html

```diff
@@ -111,10 +111,45 @@ <h2 class="card-pf-title">
                 <span translatable="yes">Your login credentials do not give you access to use the docker registry from the command line.</span>
             </div>
 <code ng-if="settings.registry.password">$ sudo docker login -p {{settings.registry.password}} -e unused -u unused <span ng-class="{ placeholder: !settings.registry.host_explicit }">{{settings.registry.host}}</span></code>
-            <p translatable="yes">Push an image:</p>
+            <p translatable="yes">Log into OpenShift:
```

Comment:
Translatable element's can't have sub elements. You need to remove translatable on the p, wrap the heading text in a span, and make the span and the a below translatable.